LEGO1/realtime/matrix4d.inl.h

```diff
@@ -349,14 +352,19 @@ int Matrix4::Invert(Matrix4& p_mat)
 		}
 	}
 
+	// SUCCESS from mxtypes.h
 	return 0;
 }
 
 // FUNCTION: LEGO1 0x1006b500
+// FUNCTION: BETA10 0x1005aa20
 void Matrix4::Swap(int p_d1, int p_d2)
 {
-	for (int i = 0; i < 4; i++) {
-		float e = m_data[p_d1][i];
+	// This function is affected by entropy even in debug builds
+	int i;
+	float e;
+	for (i = 0; i < 4; i++) {
+		e = m_data[p_d1][i];
```

Comment:
This had no meaningful impact on entropy (it was rather neutral), but it did improve the stack on BETA10.